semgrep/semgrep/rule.py

```diff
@@ -129,6 +129,10 @@ def expression(self) -> BooleanRuleExpression:
     def fix(self) -> Optional[str]:
         return self._raw.get("fix")
 
+    @property
+    def equivalences(self) -> Dict[str, str]:
```

Comment:
Would it make more sense for this to return a list of Equivalence objects?

Reply:
Probably. I mimicked how it's done for `Pattern`; which means they don't actually turn into Equivalences until later on. I'll give it another look over to see if it makes sense to use Equivalences here